### ISSUE TITLE
Some Runs Have Pixels with Unreasonably High Radiances

### ISSUE THREAD

**Issue by michaeljp24** [aluque/CloudScat.jl#4]
Certain runs (seemingly at random, but probably) produce output with extremely high radiance values. While most of the image is around 1e-10 or lower, these pixels list radiances of 1e50, 1e100, 1e200, etc.

Removing these pixels results in a reasonable image for the cloud scene (minus the holes where the pixels were).

Any idea what might be causing this behavior? Perhaps an overflow condition?

**Comment by aluque** [comment]
Yes, that looks like some kind of overflow problem, possibly in computing intersections for geometric shapes. Do you have a working example where this happens? What was the cloud geometry like?
